pkg/objstore/client/factory.go

```diff
@@ -26,6 +27,7 @@ const (
 	AZURE ObjProvider = "AZURE"
 	SWIFT ObjProvider = "SWIFT"
 	COS   ObjProvider = "COS"
+	AliYunOSS   ObjProvider = "AliYunOSS"
```

Comment:
This inconsistent. Let's either change `AliYunOSS` to `ALIYUNOSS` or  rest to be camel case (:

Reply:
From user perspective it will not matter, so it is only the internal code readability

Reply:
I think we should change to OSS, because its product name is Object Storage Service and almost everyone uses OSS as its short name.;)

Reply:
Wow, good to know, thanks for input. :+1:  

Let's however remember that it might be clear for you guys, but not necesarrly for other continents, so I would vote for explicit name. No harm in that! (: `Object Storage Service` might be quite generic name for some users.

Reply:
Thanks @bwplotka for your warm reminder. It will be better if we explain this in the documentation. For example, we can change `AliYun OSS Configuration` to `Aliyun Object Storage Service(OSS) Configuration`.